pkgs/servers/mobilizon/js.nix

```diff
@@ -1,29 +1,17 @@
-{ stdenv, lib, yarn, mkYarnPackage, mobilizon, imagemagick }:
+{ lib, applyPatches, yarn, mkYarnPackage, mobilizon, imagemagick }:
 
 mkYarnPackage rec {
-  src = stdenv.mkDerivation {
+  src = applyPatches {
     name = "mobilizon-js-src";
-
     src = "${mobilizon.src}/js";
-
     patches = [
-      # Due to the unsupported "resolution" parameter of "package.json"
-      ./fix-yarn-lock.patch
-      # I'm not sure why this doesn't work, but this is just the Vue config
-      # setting a lower limit on memory usage, it should not affect the output
-      ./fix-vue-config.patch
+      ./set-interval.patch
```

Comment:
did you try to submit the patch upstream ? Just in case they accept it so we don't need to patch anymore.